README.md

```diff
@@ -1,2 +1,2 @@
 # tinlake-math
-Safe Arithmetic, based on ds-math  https://dapp.tools/dappsys/ds-math.html
+
```

Comment:
why delete this sentence?

Reply:
because this repo contains other util functions than safe math

Reply:
then let's extend it instead :)